src/components/SubjectViewer/components/AnnotationsPane/AnnotationsPaneContainer.js

```diff
@@ -2,19 +2,42 @@ import React from 'react'
 import { observer } from 'mobx-react'
 import AppContext from 'store'
 import AnnotationsPane from './AnnotationsPane'
-import { constructCoordinates } from 'helpers/parseTranscriptionData'
+import { constructCoordinates, constructText, mapExtractsToReductions } from 'helpers/parseTranscriptionData'
 
 function AnnotationsPaneContainer({ x, y }) {
-  let parsedLines = []
+  let [ extractLines, reductionLines ] = [[], []]
```

Comment:
Is this just a fancy way of writing `let extractLines = [], reductionLines = []` ?